TPP.Core/Commands/Definitions/UserCommands.cs

```diff
@@ -59,6 +59,17 @@ public class UserCommands : ICommandCollection
                 Aliases = new[] { "gifttokens" },
                 Description = "Donate another user tokens. Arguments: <user to donate to> <amount of tokens>"
             },
+            new Command("list", List)
+            {
+                Description = "List all of the people who have a given role." +
+                              "Arguemnts: <Role>"
+            },
+            new Command("showroles", ShowRoles)
+            {
+                Aliases = new[] {"roles"},
+                Description = "Operators only: Show which roles a user has." +
+                              "Arguments: <user>"
+            }
```

Comment:
pls use trailing commas where possible to improve git diffs when people add further entries. Same goes for the list of operator commands